client/lib/email-followers/README.md

```diff
@@ -30,51 +30,53 @@ Fetches followers in batches of 100 starting from the given page, which defaults
 /**
  * External dependencies
  */
-var React = require( 'react' );
+import { Component } from 'react';
 
 /**
  * Internal dependencies
  */
-var EmailFollowersStore = require( 'lib/followers/wpcom-followers-store' ),
-	EmailFollowersActions = require( 'lib/followers/actions' );
+import EmailFollowersStore from 'lib/followers/wpcom-followers-store';
+import EmailFollowersActions from 'lib/followers/actions';
 
-module.exports = React.createClass( {
+class YourComponent extends Component {
 
-	displayName: 'yourComponent',
+	constructor( props ) {
+		super( props );
```

Comment:
I think I've seen `super( ...arguments );` argued against in Calypso PRs - I'm all for passing everything though, it seems much more generic and generic is good :)